babel/src/main/codegen/includes/parserImpls.ftl

```diff
@@ -1470,6 +1478,66 @@ SqlTableAttribute AlterTableAttributeOnCommit() :
     { return new SqlAlterTableAttributeOnCommit(getPos(), onCommitType); }
 }
 
+/**
+  * Parses a list of alter options (ex. ADD, DROP, RENAME) for
```

Comment:
nit: Javadoc style. [Google Java Style for Javadoc](https://google.github.io/styleguide/javaguide.html#s7.1.1-javadoc-multi-line)

Reply:
Assuming you mean the indentation level?

Reply:
Yes.